share_extension/ios/extension_post.js

```diff
@@ -75,7 +77,7 @@ export default class ExtensionPost extends PureComponent {
         const isLandscape = width > height;
 
         this.state = {
-            currentUserId: null,
+            entities: props.entities,
```

Comment:
Why put the entities in state instead of just continuing to read them from the props? Doesn't doing this mean that this component will never see any changes made to entities, or will that not be happening since these are just the main app's entities, not the ones loaded by the share extension?

Reply:
Ah, you're making changes to this down below. Never mind then